src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/PhysicalTableManager.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package com.salesforce.dynamodbv2.mt.mappers.sharedtable.impl;
+
+import static java.lang.String.format;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.model.CreateTableRequest;
+import com.salesforce.dynamodbv2.mt.admin.AmazonDynamoDbAdminUtils;
+import com.salesforce.dynamodbv2.mt.mappers.metadata.DynamoTableDescription;
+import com.salesforce.dynamodbv2.mt.mappers.metadata.DynamoTableDescriptionImpl;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PhysicalTableManager {
+
```

Comment:
Moved the create & describe table logic plus the physical table description cache from TableMappingFactory to here. Also have a delete table method.